src/utils/formatEventDay.js

```diff
@@ -8,7 +8,7 @@ import format from 'date-fns/format'
  */
 
 function formatDay(startDate = '') {
-  return format(new Date(startDate), 'dd')
+  return format(new Date(startDate), 'd')
```

Comment:
since this util is also used by BlockClipped date, this will impact those styles. it seems they were meant to have 2 digits. 

I think you may need to avoid using this util in BlockCardThreeColumn altogether, revert it to its previous state, and just `format(new Date(startDate), 'd')` directly in the BlockCardThreeColumn component

Reply:
I added tests to block-clipped-date and it still works fine with the single day format https://deploy-preview-638--ucla-library-storybook.netlify.app/?path=/story/block-clipped-date--date-check

Reply:
👍